enums.h

```diff
@@ -2287,6 +2287,7 @@ typedef enum _music_id {
 	NUM_MUSIC,
 } _music_id;
 
+#ifndef NONET
 typedef enum _mainmenu_selections {
 	MAINMENU_SINGLE_PLAYER = 1,
 	MAINMENU_MULTIPLAYER   = 2,

```

Comment:
NONET is still capable of running a game via loopback, palyers used to use null modem to play multipler solo, either for the deficaulity or to train there hero.

Reply:
Fixed this in my latest PR